examples/terraform-basic-example/variables.tf

```diff
@@ -26,3 +25,15 @@ variable "example2" {
   description = "Example variable 2"
   default     = ""
 }
+
+variable "example_list" {
+  description = "An example variable that is a list."
+  type        = "list"
+  default     = []
+}
+
+variable "example_map" {
+  description = "An example variable that is a map."
+  type        = "map"
+  default     = {}
+}
```

Comment:
Do we have tests for nested lists and maps?

Reply:
Attempted to add some nested lists / maps testing: 6d1e95d .